src/components/StyleGuide/Typography.mdx

```diff
@@ -141,58 +485,117 @@ used for body copy, captions and secondary captions.
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="inset.default"
+>
+  <Text size="tag" m="0">
+    Tag text
+  </Text>
+</Box>
+}
 
-|                 | JS Token       | CSS Token                              | Value      |
-| --------------- | :------------- | :------------------------------------- | :--------- |
-| **Font Size**   | `text.caption` | `var(--nypl-fontSizes-text-caption)`   | `0.875rem` |
-| **Font Weight** | `text.caption` | `var(--nypl-fontWeights-text-caption)` | `300`      |
-| **Line Height** | --             | --                                     | `1.5`      |
+#### Mini (deprecated)
 
 {
 
-<Text size="tag">Tag</Text>}
+<List type="ul">
+  <li>This style should not be used.</li>
+</List>
+}
 
-|                 | JS Token   | CSS Token                          | Value     |
-| --------------- | :--------- | :--------------------------------- | :-------- |
-| **Font Size**   | `text.tag` | `var(--nypl-fontSizes-text-tag)`   | `0.75rem` |
-| **Font Weight** | `text.tag` | `var(--nypl-fontWeights-text-tag)` | `400`     |
-| **Line Height** | --         | --                                 | `1.5`     |
+|                 | JS Token    | CSS Token                           | Value             |
+| --------------- | :---------- | :---------------------------------- | :---------------- |
+| **Font Size**   | `text.mini` | `var(--nypl-fontSizes-text-mini)`   | `0.625rem` (10px) |
+| **Font Weight** | `text.mini` | `var(--nypl-fontWeights-text-mini)` | `400` (regular)   |
+| **Line Height** | --          | --                                  | `1.5`             |
 
 {
 
-<Text size="mini">Mini</Text>}
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="inset.default"
+>
+  <Text size="mini" m="0">
+    Mini text (deprecated)
+  </Text>
+</Box>
+}
 
-|                 | JS Token    | CSS Token                           | Value      |
-| --------------- | :---------- | :---------------------------------- | :--------- |
-| **Font Size**   | `text.mini` | `var(--nypl-fontSizes-text-mini)`   | `0.625rem` |
-| **Font Weight** | `text.mini` | `var(--nypl-fontWeights-text-mini)` | `400`      |
-| **Line Height** | --          | --                                  | `1.5`      |
+#### Subtitle 1
 
-## Font Family
+{
 
-With website accessibility in mind, font choices for the DS were made with an
-eye toward clarity and readability. Headings and body content use the
-following font family assignment:
+<List type="ul">
+  <li>
+    This style SHOULD be used with text set as a subtitle grouped with a heading
+    element.
+  </li>
+  <li>Text set in this style MAY be sentence length.</li>
+</List>
+}
+
+|                 | JS Token   | CSS Token                          | Value             |
+| --------------- | :--------- | :--------------------------------- | :---------------- |
+| **Font Size**   | `text.tag` | `var(--nypl-fontSizes-text-tag)`   | `1.125rem` (18px) |
```

Comment:
Subtitle1 and subtitle2 tables have the same JS token and CSS token values in the table. From the description, it's still not clear when to use subtitle1 over subtitle2